src/SFA.DAS.EmployerDemand.Domain.UnitTests/Demand/Api/WhenBuildingTheGetProviderEmployerDemand.cs

```diff
@@ -37,7 +39,28 @@ public void Then_The_Url_Is_Correctly_Built_With_Location_Filter(int ukprn, stri
             var actual = new GetProviderEmployerDemandRequest(ukprn,null, locationParam, locationRadius);
             
             //Assert
-            actual.GetUrl.Should().Be($"demand/aggregated/providers/{ukprn}?courseId=&location={HttpUtility.UrlEncode(locationParam)}&locationRadius={locationRadius}");
+            actual.GetUrl.Should().Be($"demand/aggregated/providers/{ukprn}?courseId=&location={HttpUtility.UrlEncode(locationParam)}&locationRadius={locationRadius}&sectors=");
+        }
+
+        [Test, AutoData]
+        public void Then_The_Url_Is_Correctly_Built_With_Routes(int ukprn, List<string> selectedRoutes)
+        {
+            //Act
+            var actual = new GetProviderEmployerDemandRequest(ukprn,null, null, null, selectedRoutes);
+            
+            //Assert
+            actual.GetUrl.Should().Be($"demand/aggregated/providers/{ukprn}?courseId=&location=&locationRadius=&sectors=" 
+                                      + string.Join("&sectors=", selectedRoutes.Select(HttpUtility.UrlEncode)));
+        }
+
+        [Test, AutoData]
+        public void Then_If_There_Is_A_SelectedCourse_And_Routes_Only_SelectedCourse_Is_Added(int ukprn, List<string> selectedRoutes, int courseId)
```

Comment:
this logic is also present in the handler for the response to the controller (I'm guessing to clear the check boxes on the UI). so is this needed as well????